test/jdk/javax/swing/text/html/FormView/4473401/frameset.html

```diff
@@ -0,0 +1,11 @@
+<HTML>
+ <HEAD>
+   <TITLE>
+     Manual test for bug 4463014
+   </TITLE>
+ </HEAD>
+ <FRAMESET COLS="150,*">
+  <FRAME SRC="frame1.html">
+  <FRAME name="main" SRC="frame2.html">
+ </FRAMESET>
+</HTML>
```

Comment:
Having a blank line in the end of the file, that is ending the file with `\n`, is recommended, though not required.